doc/variable.md

```diff
@@ -11,36 +11,29 @@ for Python interface is transposed.
 **Attention**: In Python interface, `lattice` is transposed
 ({ref}`py_variables_crystal_structure`).
 
-Basis vectors (in Cartesian)
+Basis vectors (in Cartesian coordinates)
 
 ```
 [ [ a_x, b_x, c_x ],
   [ a_y, b_y, c_y ],
   [ a_z, b_z, c_z ] ]
 ```
 
-Cartesian position $\mathbf{x}_\mathrm{Cart}$ is obtained by
-
-$$
-  \mathbf{x}_\mathrm{Cart} = \mathrm{L}\cdot\mathbf{x}_\mathrm{frac}
-$$
-
-where $\mathrm{L}$ is the basis vectors defined above and is
-$\mathrm{L}=(\mathbf{a},\mathbf{b},\mathbf{c})$, and
-$\mathbf{x}_\mathrm{frac}$ is the atomic position in fractional
-coordinates given below.
+See {ref}`basis vectors <def_basis_vectors>` and {ref}`point coordinates <def_point_coordinates>` for more details.
```

Comment:
Minor note, you can use markdown linkage.
```suggestion
See [basis vectors](../definition.md#def_basis_vectors) and [point coordinates](../definition.md#def_point_coordinates) for more details.
```

Reply:
Thanks. I updated to cross-referencing as shown in d1103a1. The way `../definition.md` didn't work on my computer setting, so I dropped those.

Reply:
Good to know. That must have been because a global anchor was used